server/src/main/scala/core/Core.scala

```diff
@@ -23,7 +23,7 @@ import akka.actor.ActorSystem
  */
 trait Core {
 
-  implicit def actorSystem: ActorSystem
```

Comment:
It's good practice to make your traits declare defs... that way your implementing classes can be vars or defs, but the opposite is not true.